### PR TITLE
[eclipse/xtext#1765] update to MWE 2.11.3.M3

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1765] update to MWE 2.11.3.M3
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="org.eclipse.emf.mwe2.language.sdk"
-      version="2.11.2.v20200224-0816"
+      version="2.11.3.v20200224-0816"
```
Comment (szarnekow) [inline releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml:4]: The timestamp looks weird.

File: releng/org.eclipse.emf.mwe2.language.sdk.dummy/pom.xml [hunk 2]
```diff
@@ -10,5 +10,5 @@
 	<packaging>eclipse-feature</packaging>
 	<groupId>org.eclipse.emf</groupId>
 	<artifactId>org.eclipse.emf.mwe2.language.sdk</artifactId>
-	<version>2.11.2.v20200224-0816</version>
+	<version>2.11.3.v20200224-0816</version>
```
Comment (szarnekow) [inline releng/org.eclipse.emf.mwe2.language.sdk.dummy/pom.xml:13]: Same here

File: releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml (hunk 1 above)
Comment (cdietrich) [reply releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml:4]: is random.
will be chaned to real on on release only

File: releng/org.eclipse.emf.mwe2.language.sdk.dummy/pom.xml (hunk 2 above)
Comment (cdietrich) [reply releng/org.eclipse.emf.mwe2.language.sdk.dummy/pom.xml:13]: this is just a random timestamp to satisfy tycho.

**Pull request merged**